packages/collections/graphql/schema.js

```diff
@@ -13,5 +13,16 @@ type mutations {
     documentId: ID!
     collectionName: String!
   ): CollectionItem
+
+
+  upsertDocumentProgress(
+    documentId: ID!
+    percentage: Int!
+    nodeId: ID!
```

Comment:
`nodeId` is String rather than an ID I assume.

Reply:
it's actually the same thing, but yes `String` is more accurate.